### ISSUE TITLE
TNB List - Design - 2

### ISSUE THREAD

**Issue by avinashupadhya99** [thenewboston-developers/Projects#246]
#### Link to original project proposal.
#170 

#### Link to most recent milestone payout request.
#243 

#### Milestone Description
SHORT DESCRIPTION OF MILESTONE HERE

| # | Objective             | How can auditors verify milestone?     |
| - | --------------------- | -------------------------------------- |
| 1 | Create a Figma design for the UI | Check the Figma |
| 2 | Create the complete backend architecture along with all the microservices | Document with the architecture |
| 3 | Setup the repos for all microservices | Check Github |
| 4 | Implement the landing page UI | Check website |

#### How long do you estimate this milestone will take to complete?
1-2 weeks

#### Project Wallet
`94cf53921e9c5ddf48dc5589b7cdf111f7acdc93dc3073b94880416d4676b9ae`

**Comment by tspearing** [comment]
Hi @avinashupadhya99, regarding objectives 1,2 and 4, please indicate the level of detail planned. Are the UI's pixel perfect hifi? What level of backend architecture will be detailed? Point form notes or something much more in depth?

**Comment by avinashupadhya99** [comment]
Yes, 1 will be a pixel perfect UI with Figma. The current UI wireframe does not have the color and is not a polished design. The backend architecture will be a complete design of the backend with all the microservices involved, database, the messaging system and the cache if necessary. The landing page will also be pixel perfect and will aim for responsiveness as well.

**Comment by tspearing** [comment]
Hi @avinashupadhya99, we approve this for 8500 TNBC.

**Comment by tspearing** [comment]
Hey @avinashupadhya99, just checking in to see how this project is coming along. Let us know if you need anything.
